Foundation/NSJSONSerialization.swift

```diff
@@ -159,7 +159,13 @@ public class JSONSerialization : NSObject {
     /* Write JSON data into a stream. The stream should be opened and configured. The return value is the number of bytes written to the stream, or 0 on error. All other behavior of this method is the same as the dataWithJSONObject:options:error: method.
      */
     public class func writeJSONObject(_ obj: AnyObject, toStream stream: NSOutputStream, options opt: WritingOptions) throws -> Int {
-        NSUnimplemented()
+        do {
```

Comment:
This API differs from the Darwin version,the latter takes a fourth  argument of type ```NSErrorPointer``` which is  a typealias unavailable on Linux .The Darwin version also does not throw any error.The spec says that 0 needs to be returned on error.If we return 0 we cannot throw the error.